tests/testthat/test-fetchGeoFile.R

```diff
@@ -9,27 +9,29 @@ with_mock_crunch({
             geodatum = "https://app.crunch.io/api/geodata/newone/",
             feature_key = "none",
             match_field = "none")
-        expect_error(fetchGeoFile(test_crGeo), paste("Unknown format", dQuote("notjson"), "in geodata url:", "https://s.crunch.io/some/wrong/path.geojson"))
+        expect_error(fetchGeoFile(test_crGeo), 
+                     paste("Unknown format", dQuote("notjson"), "in geodata url:", 
+                           "https://s.crunch.io/some/wrong/path.geojson"))
     })
 
     test_that("fetchGeoFile works with other arguments", {
-        expect_error(fetchGeoFile("foo"),
-                     "Cannot fetch a geography on objects other than" ,
-                     " CrunchGeography or CrunchVaraibles.")
+        expect_error(fetchGeoFile("foo"), paste0(
+            "Cannot fetch a geography on objects other than ",
+            "CrunchGeography or CrunchVaraibles."))
     })
 
     test_that("getGeoDataFrame errors", {
-        expect_GET(fetchGeoFile(ds$location),
+        # Can't use `expect_GET` because `topojson_read()` doesn't use httr
+        expect_error(suppressWarnings(fetchGeoFile(ds$location)),
```

Comment:
Want to make sure I'm understanding httptest correctly. If I use `expect_GET` I get a failing test that says: 

```r
test-fetchGeoFile.R:25: failure: getGeoDataFrame errors
`object` threw an error with unexpected message.
Expected match: "GET https://s.crunch.io/some/wrong/gb_eer_doesnotexist.topojson"
Actual message: "Cannot open \"https://s.crunch.io/some/wrong/gb_eer_doesnotexist.topojson\"; The file doesn't seem to exist."
```

Is there a better way to do this with `httptest`? If not, do you want me to be more specific with my error message? Currently this works because it looks for the url anywhere in the message.

Reply:
I am sure there is, let me find examples. There are a number of test expectation helpers in rcrunch, but possibly some baked into httptest.